code/modules/mining/mine_items.dm

```diff
@@ -270,7 +270,7 @@ proc/move_mining_shuttle()
 
 /obj/item/weapon/pickaxe/plasmacutter/accelerator
 	name = "plasma cutter"
-	desc = "A rock cutter that fires bolts of hot plasma. You could use it to cut limbs off of xenos! Or, you know, mine stuff."
```

Comment:
"A rock cutter that utilizes plasma to create sharp beams of energy. The cutting power is enough to cut through rocks and xenos with ease. It requires solid plasma to function.

Reply:
I preferred the little sillyness of the old one.